README.md

```
@@ -9,115 +1,60 @@
 
-  <a href="https://github.com/themefisher/pinwheel-astro/blob/main/LICENSE">
-    <img src="https://img.shields.io/github/license/themefisher/pinwheel-astro" alt="license"></a>
+## AMRIT (Accessible Medical Records via Integrated Technologies)
```

Markdownlint (0.35.0)

First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)